cmd/metrics-v3-types.go

```diff
@@ -267,7 +267,7 @@ func (m *MetricValues) SetHistogram(name MetricName, hist *prometheus.HistogramV
 		panic(fmt.Sprintf("metric has no description: %s", name))
 	}
 	dummyDesc := MetricDescription{}
-	metricsV2 := getHistogramMetrics(hist, dummyDesc, false)
+	metricsV2 := getHistogramMetrics(hist, dummyDesc, false, false)
```

Comment:
@klauspost @anjalshireesh  I'm not sure why this V2 functionality is invoked by V3 metrics. Will it be limited by this as well. I haven't tested this case as yet.

Reply:
Tested all V3 metrics as well. They are not affected.